Jenkinsfile

```diff
@@ -257,6 +257,8 @@ pipeline {
                         runTest('arbiter', 'basic')
                         runTest('service-per-pod', 'basic')
                         runTest('liveness', 'basic')
+                        runTest('smart-update', 'basic')
```

Comment:
please add these tests to all needed jenkins jobs `psmdb_operator_*.groovy` too
https://github.com/Percona-Lab/jenkins-pipelines/tree/master/cloud/jenkins

Reply:
@hors, done
https://github.com/Percona-Lab/jenkins-pipelines/pull/559